src/library/Cardano/Fee.hs

```diff
@@ -102,7 +99,8 @@ newtype Fee = Fee
 
 -- | Defines the maximum size of a dust coin.
 --
--- See function 'dustThreshold' within 'FeeOptions'.
+-- Change values that are less than or equal to this threshold will be
+-- evicted from created transactions.
```

Comment:
you mean changes dusted will not participate in creating the transaction, right?

Reply:
> you mean changes dusted will not participate in creating the transaction, right?

I agree, this wording is a little awkward. Fixed in: 373a41ac334b5e88e51f6fcbd1b60ed66f31432b